README.md

```diff
@@ -27,4 +27,19 @@ cd jasyncapi-idea-plugin
+| [1.4.0+idea2021](https://github.com/asyncapi/jasyncapi-idea-plugin/releases/tag/1.4.0%2Bidea2021) | Bumblebee &#124; 2021.1.1 — Dolphin &#124; 2021.3.1 Patch 1  | 2021.1 — 2021.3.3 | build 211.0 — 213.* | 2021.1 — 2021.3.4 | build 211.0 — 213.* | 2021.1 — 2021.3.4 | 2021.3 — 2021.3.3   | 2021.1 — 2021.3.5 | 2021.1 — 2021.3.3       | 2021.1 — 2021.3.4         | 2021.1 — 2021.3.3      | build 211.0 — 213.* | 2021.1 — 2021.3.2 | 2021.1 — 2021.3.3 | 2021.1 — 2021.3.3 | 2021.1 — 2021.3.4   | 2021.1 — 2021.3.3    | 2021.1 — 2021.3.4 | 2021.1 — 2021.3.3 | 2021.1 — 2021.3.3 |
+| [1.5.0+idea2021](https://github.com/asyncapi/jasyncapi-idea-plugin/releases/tag/1.5.0%2Bidea2021) | Bumblebee &#124; 2021.1.1 — Electric Eel &#124; 2022.1.1     | 2021.1 — 2022.1.4 | build 211.0 — 221.* | 2021.1 — 2022.1.3 | build 211.0 — 221.* | 2021.1 — 2022.1.5 | 2021.3 — 2022.1.4   | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.4       | 2021.1 — 2022.1.3         | 2021.1 — 2022.1.4      | build 211.0 — 221.* | 2021.1 — 2021.3.2 | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.3   | 2021.1 — 2022.1.4    | 2021.1 — 2022.1.2 | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.4 |
+| [1.6.0+idea2021](https://github.com/asyncapi/jasyncapi-idea-plugin/releases/tag/1.6.0%2Bidea2021) | Bumblebee &#124; 2021.1.1 — Electric Eel &#124; 2022.1.1     | 2021.1 — 2022.1.4 | build 211.0 — 221.* | 2021.1 — 2022.1.3 | build 211.0 — 221.* | 2021.1 — 2022.1.5 | 2021.3 — 2022.1.4   | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.4       | 2021.1 — 2022.1.3         | 2021.1 — 2022.1.4      | build 211.0 — 221.* | 2021.1 — 2021.3.2 | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.3   | 2021.1 — 2022.1.4    | 2021.1 — 2022.1.2 | 2021.1 — 2022.1.4 | 2021.1 — 2022.1.4 |
+| [1.7.0+idea2021](https://github.com/asyncapi/jasyncapi-idea-plugin/releases/tag/1.7.0%2Bidea2021) | Bumblebee &#124; 2021.1.1 — Flamingo &#124; 2022.2.1 Beta 1  | 2021.1 — 2022.2.5 | build 211.0 — 222.* | 2021.1 — 2022.2.4 | build 211.0 — 222.* | 2021.1 — 2022.2.5 | 2021.3 — 2022.2.4   | 2021.1 — 2022.2.5 | 2021.1 — 2022.2.4       | 2021.1 — 2022.2.2         | 2021.1 — 2022.2.4      | build 211.0 — 222.* | 2021.1 — 2022.2   | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.2   | 2021.1 — 2022.2.4    | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.4 |
+| [1.7.1+idea2021](https://github.com/asyncapi/jasyncapi-idea-plugin/releases/tag/1.7.1%2Bidea2021) | Bumblebee &#124; 2021.1.1 — Flamingo &#124; 2022.2.1 Beta 1  | 2021.1 — 2022.2.5 | build 211.0 — 222.* | 2021.1 — 2022.2.4 | build 211.0 — 222.* | 2021.1 — 2022.2.5 | 2021.3 — 2022.2.4   | 2021.1 — 2022.2.5 | 2021.1 — 2022.2.4       | 2021.1 — 2022.2.2         | 2021.1 — 2022.2.4      | build 211.0 — 222.* | 2021.1 — 2022.2   | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.2   | 2021.1 — 2022.2.4    | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.4 | 2021.1 — 2022.2.4 |
```

Comment:
@Pakisan  if this is for a `v1.8.0` release, shouldn't this table include a row for `1.8.0+idea2022` or `1.8.0+jre11`?

Reply:
I Added 2.0.0+jre17, 1.8.0+jre11